modules/jenkins_files/files/var/lib/jenkins/scriptApproval.xml

```diff
@@ -201,4 +204,4 @@
   <pendingScripts/>
   <pendingSignatures/>
   <pendingClasspathEntries/>
-</scriptApproval>
+</scriptApproval>
```

Comment:
This change should be avoided.

Reply:
Can you please elaborate on why? I took the file directly off the running host so this should be exactly how Jenkins wrote it to disk.

Reply:
Usually we try to avoid pure white space changes. I don't know why the trailing newline was there in the first place. Your previous [commit](https://github.com/ros-infrastructure/buildfarm_deployment/commit/f77918f6f042ca9362797d1782d98bdf56d69270#diff-ff43ec85b9b332437c9ad8f86c2cf7d1) added it. We should use whatever the "normative" behavior of Jenkins is - which ever of the two options that is.

Reply:
That looks like my mistake previously. My editor enforces trailing newlines by default. I'll have to add an override for this file and other common Jenkins ones.